test/e2e/network/dns_configmap.go

```diff
@@ -193,3 +193,46 @@ var _ = SIGDescribe("DNS configMap nameserver", func() {
 		t.run()
 	})
 })
+
+type dnsPtrFwdTest struct {
+	dnsTestCommon
+}
+
+func (t *dnsPtrFwdTest) run() {
+	t.init()
+
+	t.createUtilPod()
+	defer t.deleteUtilPod()
+
+	t.createDNSServerWithPtrRecord()
+	defer t.deleteDNSServerPod()
+
+	t.setConfigMap(&v1.ConfigMap{Data: map[string]string{
+		"stubDomains":         fmt.Sprintf(`{"acme.local":["%v"]}`, t.dnsServerPod.Status.PodIP),
```

Comment:
Seems like this stubdomain setup is not needed?

Reply:
Removed.